### PR TITLE
4 services fixed - new approach

### PR DESCRIPTION
**Pull request by highlandr**
LegendasDivx, LegendasZone, Pipocas and euTorrents.
Tested on linux, ios and windows.

### CODE REVIEW

File: script.xbmc.subtitles/resources/lib/services/LegendasDivx/service.py [hunk 1]
```diff
@@ -71,10 +74,8 @@
 _ = sys.modules[ "__main__" ].__language__
 __scriptname__ = sys.modules[ "__main__" ].__scriptname__
 __addon__ = sys.modules[ "__main__" ].__addon__
-__cwd__        = sys.modules[ "__main__" ].__cwd__
+__cwd__        = sys.modules[ "__main__" ].__cwd__.decode('utf-8')
```
Comment (jurrabi) [inline script.xbmc.subtitles/resources/lib/services/LegendasDivx/service.py:77]: If you check default.py **cwd** is already decoded.
default.py:
(16) **cwd**        = xbmc.translatePath( **addon**.getAddonInfo('path') ).decode("utf-8")

So, no need for this (I think).

Comment (highlandr) [inline script.xbmc.subtitles/resources/lib/services/LegendasDivx/service.py:77]: didn't check but I believe you :+1:

Comment (amet) [inline script.xbmc.subtitles/resources/lib/services/LegendasDivx/service.py:77]: can you revert it and try please? let me know if its ok and we can cleanup here then

Comment (highlandr) [inline script.xbmc.subtitles/resources/lib/services/LegendasDivx/service.py:77]: It is in the default.py
**cwd**        = xbmc.translatePath( **addon**.getAddonInfo('path') ).decode("utf-8")

redundancy? :)

File: script.xbmc.subtitles/resources/lib/services/LegendasDivx/service.py [hunk 2]
```diff
@@ -130,7 +131,7 @@
 #====================================================================================================================
 def msgnote(site, text, timeout):
```
Comment (jurrabi) [inline script.xbmc.subtitles/resources/lib/services/LegendasDivx/service.py:132]: I see you didn't added the isinstance checks. Is this working for you in all cases? Don't have xbmc at work to check.

File: script.xbmc.subtitles/resources/lib/services/LegendasDivx/service.py (hunk 1 above)
Comment (jurrabi) [inline script.xbmc.subtitles/resources/lib/services/LegendasDivx/service.py:77]: Yes. It is redundant. You can take it out.

Comment (highlandr) [inline script.xbmc.subtitles/resources/lib/services/LegendasDivx/service.py:77]: Ok let me get my hands on the laptop.

In 30 minutes driving to Datacenter :)

**Pull request merged**